docs/discovery.md

```diff
@@ -194,7 +194,7 @@ swarm is connected to the public internet. To create your cluster:
 
     This can be on any machine or even your laptop.
 
-        $ swarm manage -H tcp://<swarm_ip:swarm_port> token://<cluster_id>
+        $ swarm manage -H <swarm_ip:swarm_port> token://<cluster_id>
```

Comment:
I am afraid not. Only you can update your PR with commit handling. So, squash commits is necessary.